### ISSUE TITLE
Can't fully turn on ActivityPub - remote interactions seem broken

### ISSUE THREAD

**Issue by robertpi** [pixelfed/pixelfed#3959]
Hello,

I've got an pixelfed instance up and running and locally everything seems to work well, but all of the federation features seem broken. 

I can find remote users and follow them, but I don't see there activity and they don't show up in my follow count.

Remote uses can find and follow an account on my instance, but afterwards that account is not shown in their following list and the accounts activity doesn't show in the timeline.

I've noticed that the ActivityPub variables in diagnostics  do not seem to match what I have in my `.env` file.

![Screenshot 2022-12-18 at 07-36-36 Pixelfed](https://user-images.githubusercontent.com/51381/208285250-e525b3e9-a09e-4c9e-af41-d4f52b85641a.png)

```
# ActivityPub Configuration
ACTIVITY_PUB="true"
AP_REMOTE_FOLLOW="true"
AP_INBOX="true"
AP_OUTBOX="true"
AP_SHAREDINBOX="true"
```

I've tried running `php artisan config:cache` & `php artisan config:clear` to get the file and the diagnostic variables to sync but it doesn't seem to make a difference for these settings. (Changing other settings like `CUSTOM_EMOJI` does work)

I'm running the dev branch, as suggested in the installation guide, the last commit I have is:

```
commit b9daa3fb483ffedacb81b33e52d86260802309e8 (HEAD -> dev, origin/dev, origin/HEAD)
Merge: 8bd1b538 f85e4843
Author: daniel
Date:   Fri Dec 16 00:53:18 2022 -0700

    Merge pull request #3944 from pixelfed/staging

    Add optional home feed caching
```
My system information is:

![Screenshot_20221218_073854](https://user-images.githubusercontent.com/51381/208285270-541a53fe-604e-4ffb-a068-9b4120e0fa4c.png)

Any suggestions gratefully received. Many thanks.

**Comment by win0err** [comment]
Maybe this happens due to outbox is disabled: https://github.com/pixelfed/pixelfed/issues/3906

**Comment by dansup** [comment]
@robertpi Please join our discord and ping me, [discord.gg/MHvDHaSzmc](https://discord.gg/MHvDHaSzmc)

**Comment by robertpi** [comment]
After following up on discord, it seems to be fixed now.

I'd already checked Settings > ActivityPub, because I realized I hadn't set it, when I created the original env file. This seemed to mean changes in the .env file where ignored.

Unchecking it saving and checking it again seemed to make the other setting activate

Difficult to be certain that's what fixed it, as there was something of a gap in my testing, so it could have just been some cache catching up.

Anyway, happy it's fixed.
